mail_tracking_mailgun/models/mail_tracking_email.py

```diff
@@ -0,0 +1,196 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Antonio Espinosa - <antonio.espinosa@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import hashlib
+import hmac
+import threading
+from datetime import datetime
+from openerp import models, api, fields
+
+import logging
+_logger = logging.getLogger(__name__)
+
+
+class MailTrackingEmail(models.Model):
+    _inherit = "mail.tracking.email"
+
+    def _country_search(self, country_code):
+        country = False
+        if country_code:
+            country = self.env['res.country'].search([
+                ('code', '=ilike', country_code),
```

Comment:
IMO `=` would be a better operator here, just `country_code..capitalize()` for the casing